geoapps/inversion/natural_sources/tipper/constants.py

```diff
@@ -0,0 +1,364 @@
+    "detrend_order": None,
+    "detrend_type": None,
+    "max_chunk_size": 128,
+    "chunk_by_rows": True,
+    "mesh": None,
+    "u_cell_size": 25.0,
+    "v_cell_size": 25.0,
+    "w_cell_size": 25.0,
+    "octree_levels_topo": [0, 0, 4, 4],
+    "octree_levels_obs": [4, 4, 4, 4],
+    "depth_core": 500.0,
+    "max_distance": 5000.0,
+    "horizontal_padding": 1000.0,
+    "vertical_padding": 1000.0,
+    "window_center_x": None,
+    "window_center_y": None,
+    "window_width": None,
+    "window_height": None,
+    "window_azimuth": None,
+    "inversion_style": "voxel",
+    "chi_factor": 1.0,
+    "sens_wts_threshold": 60.0,
+    "every_iteration_bool": False,
+    "f_min_change": 1e-4,
+    "minGNiter": 1,
+    "beta_tol": 0.5,
+    "prctile": 95,
+    "coolingRate": 1,
+    "coolEps_q": True,
+    "coolEpsFact": 1.2,
+    "beta_search": False,
+    "starting_chi_factor": None,
+    "max_iterations": 25,
+    "max_line_search_iterations": 20,
+    "max_cg_iterations": 30,
+    "max_global_iterations": 100,
+    "initial_beta_ratio": 1e2,
+    "initial_beta": None,
+    "tol_cg": 1e-4,
+    "alpha_s": 1.0,
+    "alpha_x": 1.0,
+    "alpha_y": 1.0,
+    "alpha_z": 1.0,
+    "s_norm": 0.0,
+    "x_norm": 2.0,
+    "y_norm": 2.0,
+    "z_norm": 2.0,
+    "reference_model_object": None,
+    "reference_model": 1e-3,
+    "gradient_type": "total",
+    "lower_bound_object": None,
+    "lower_bound": None,
+    "upper_bound_object": None,
+    "upper_bound": None,
+    "parallelized": True,
+    "n_cpu": None,
+    "max_ram": None,
+    "out_group": "TipperInversion",
+    "monitoring_directory": None,
+    "workspace_geoh5": None,
+    "run_command": "geoapps.inversion.driver",
+    "run_command_boolean": False,
+    "conda_environment": "geoapps",
+    "distributed_workers": None,
+    "txz_real_channel_bool": False,
+    "txz_imag_channel_bool": False,
+    "tyz_real_channel_bool": False,
+    "tyz_imag_channel_bool": False,
+}
+
+forward_defaults = {
+    "title": "SimPEG Tipper Forward",
+    "inversion_type": "tipper",
+    "geoh5": None,  # Must remain at top of list for notebook app initialization
+    "forward_only": True,
+    "topography_object": None,
+    "topography": None,
+    "data_object": None,
+    "txz_real_channel_bool": False,
+    "txz_imag_channel_bool": False,
+    "tyz_real_channel_bool": False,
+    "tyz_imag_channel_bool": False,
+    "starting_model_object": None,
+    "starting_model": 1e-3,
+    "background_conductivity": 1e-3,
+    "tile_spatial": 1,
+    "output_tile_files": False,
+    "z_from_topo": False,
+    "receivers_radar_drape": None,
+    "receivers_offset_x": 0.0,
+    "receivers_offset_y": 0.0,
+    "receivers_offset_z": 0.0,
+    "gps_receivers_offset": None,
+    "resolution": None,
+    "max_chunk_size": 128,
+    "chunk_by_rows": True,
+    "mesh": None,
+    "u_cell_size": 25.0,
+    "v_cell_size": 25.0,
+    "w_cell_size": 25.0,
+    "octree_levels_topo": [16, 8, 4, 2],
```

Comment:
[0, 0, 4, 4]?

Reply:
Definite no. We want to refine tight on the obs, but care less about topo. Topography is the most expensive distretization.

Reply:
This was a suggestion for topo.  instead of 16, 8, 4, 2.

Reply:
Oh yep

Reply:
Done